Complete zh-cn translation, manually translated & merged with #668

metameqa opened the pull request:
Complete zh-cn translation, manually translated by native Chinese speaker. Besides, this is not just another zh-cn translation, but rather an attempt to maximizing user experience for the Chinese users. 

Merged with @StoneMoe #668

To Chinese collaborators:
翻译说明：
1、此版文案翻译优先从Neos产品化角度考虑，尤其对于一些关键词汇的翻译。Neos团队及核心用户群都把Neos定义为一个有几乎无限自由度的创作+体验平台，即真正的Metaverse，所以几经斟酌，作者还是采用了较为创新的用词，如：
- 主菜单Tab中使用“元宇宙”，意指该Tab下包含众多星球与平行时空；
- world翻译为“星球”，一方面取自Neos中world的球状显示，另一方面，与session“时空”相比，更倾向体现world的实体性；
- session翻译为“时空”，时空是活的，一个world可以有多个平行“时空”，每个用户都可以创建同一个“星球”的不同“时空”；
- avatar翻译为“化身”，而且当3D模型没有被装备时，叫做“模型”，当模型被装备后，叫做“化身”。

2、文案翻译会结合文案在UI界面上的显示布局，如从文档中可以看到有些字段中似乎有多余的空格，原因是在Neos UI中有些按钮的形状使得文案双排显示效果更佳，因此加入“ ”空格是文案换行的方法。

3、此外还有一些UI文案标准，如括号的使用，按照主流UX规范，按钮、标题等UI元素中使用英文括号()，而大段中文说明文字中使用中文括号（），此类规范将会逐步补充，以达到合作者的统一。

metameqa committed:
Complete manual translation for Chinese zh-cn

The Neos zh-cn translation was clearly not done by native Chinese speakers (but thanks them as well). After 5 months of working on it, I've completed the manual translation of almost all keywords to Chinese (zh-cn).

StoneMoe commented:
Oh I thought this PR only translate some manual text, didn't expect it means manually translating XD
Is this PR still actively updating? Hope we can work together as a team to make zh-cn full translation merging soon.

metameqa commented:
> Oh I thought this PR only translate some manual text, didn't expect it means manually translating XD Is this PR still actively updating? Hope we can work together as a team to make a full zh-cn translation happen soon.

Great to see fellow contributors to Neos! Let's team up! Please contact me by Wechat: MetaCL or CL in Neos

metameqa committed:
zh-cn: Update & merge 2 translations

Merged with @StoneMoe

This is not just another zh-cn translation, but rather an attempt to maximizing user experience for the Chinese users.

To Chinese collaborators:
翻译说明：
1、此版文案翻译优先从Neos产品化角度考虑，尤其对于一些关键词汇的翻译。Neos团队及核心用户群都把Neos定义为一个有几乎无限自由度的创作+体验平台，即真正的Metaverse，所以几经斟酌，作者还是采用了较为创新的用词，如：
- 主菜单Tab中使用“元宇宙”，意指该Tab下包含众多星球与平行时空；
- world翻译为“星球”，一方面取自Neos中world的球状显示，另一方面，与session“时空”相比，更倾向体现world的实体性；
- session翻译为“时空”，时空是活的，一个world可以有多个平行“时空”，每个用户都可以创建同一个“星球”的不同“时空”；
- avatar翻译为“化身”，而且当3D模型没有被装备时，叫做“模型”，当模型被装备后，叫做“化身”。
2、文案翻译会结合文案在UI界面上的显示布局，如从文档中可以看到有些字段中似乎有多余的空格，原因是在Neos UI中有些按钮的形状使得文案双排显示效果更佳，因此加入“ ”空格是文案换行的方法。

metameqa committed:
Update README.md

metameqa commented:
@ProbablePrime 
I've done the merge (with #668 by @StoneMoe ), plus added some UI translation guidelines in the desc for ease of collaboration with future contributors. Would love to see Neos restarts updating some time.
